README.md

```diff
@@ -1,68 +1,45 @@
 
-## Generated Samples
 
-| Dataset       | Samples |
-| :----- |:-------------|
-| LJ      | [50k](https://soundcloud.com/kyubyong-park/sets/dc_tts) [200k](https://soundcloud.com/kyubyong-park/sets/dc_tts_lj_200k) [310k](https://soundcloud.com/kyubyong-park/sets/dc_tts_lj_310k) [800k](https://soundcloud.com/kyubyong-park/sets/dc_tts_lj_800k)|
-| Nick      | [40k](https://soundcloud.com/kyubyong-park/sets/dc_tts_nick_40k) [170k](https://soundcloud.com/kyubyong-park/sets/dc_tts_nick_170k) [300k](https://soundcloud.com/kyubyong-park/sets/dc_tts_nick_300k) [800k](https://soundcloud.com/kyubyong-park/sets/dc_tts_nick_800k)|
-| Kate| [40k](https://soundcloud.com/kyubyong-park/sets/dc_tts_kate_40k) [160k](https://soundcloud.com/kyubyong-park/sets/dc_tts_kate_160k) [300k](https://soundcloud.com/kyubyong-park/sets/dc_tts_kate_300k) [800k](https://soundcloud.com/kyubyong-park/sets/dc_tts_kate_800k) |
-| KSS| [400k](https://soundcloud.com/kyubyong-park/sets/dc_tts_ko_400k) |
+## Results
 
-## Pretrained Model for LJ
-
-Download [this](https://www.dropbox.com/s/1oyipstjxh2n5wo/LJ_logdir.tar?dl=0).
-
-## Notes
-
-  * The paper didn't mention normalization, but without normalization I couldn't get it to work. So I added layer normalization.
-  * The paper fixed the learning rate to 0.001, but it didn't work for me. So I decayed it.
-  * I tried to train Text2Mel and SSRN simultaneously, but it didn't work. I guess separating those two networks mitigates the burden of training.
-  * The authors claimed that the model can be trained within a day, but unfortunately the luck was not mine. However obviously this is much fater than Tacotron as it uses only convolution layers.
-  * Thanks to the guided attention, the attention plot looks monotonic almost from the beginning. I guess this seems to hold the aligment tight so it won't lose track.
-  * The paper didn't mention dropouts. I applied them as I believe it helps for regularization.
-  * Check also other TTS models such as [Tacotron](https://github.com/kyubyong/tacotron) and [Deep Voice 3](https://github.com/kyubyong/deepvoice3).
-  
+The code is working as it is supposed to. It creates .wav files for each sentence that was specified. The sentences can then be played using a widget. One challenge we encountered was having to change lines in hyperparams.py in order to load the right directories, since the code was most likely not intended to run in a Google Colab environment. The model dealt well with general English input, but struggled with foreign words and (proper) names. For example, the TTS model is barely able to produce a proper pronunciation of a typical Dutch name like Wouter. We did expect this to occur though, because the model is specifically trained with English language data. Further, slight glitches occurred in the audio sometimes. We encountered squeaky noises and some crackling sounds that were not part of the speech itself.

```

Comment:
It is nice that you figured out a way to automatically adapt the hyperparams.py file, however you could also have changed the directories within the synthesize.py file, in order for it to be able to run in a Google Colab environment. This is not something major however, because it doesn't matter which file you change. It's great that you found a way to make it work and even automated it. Good job!

Maybe add a discussion/findings header and create a clearer separation between the two. 
Furthermore, you could elaborate a bit on when you encountered the squeaky noises and crackling sounds and which sentences you used (maybe even the hardware?), so other people can try to replicate your findings.

Reply:
We decided to stick with changing hyperparams.py. That way if later on the user decides to train a new model, the code can be reused without issues.
+ Added a new header "findings"
+ elaborated on sound glitches found

Thanks for the feedback!

Reply:
Great job on implementing the feedback.
The only thing you could still improve is adding a specific example of a sentence where you encountered the squeaky noises and crackling sounds and specify roughly where in the sentence the sounds appear.

Reply:
added examples!

Reply:
Perfect!